controllers/vspheremachine_controller.go

```diff
@@ -191,12 +191,12 @@ func (r *machineReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_
 	}
 
 	// Fetch the CAPI Machine and CAPI Cluster.
-	machine, err := clusterutilv1.GetOwnerMachine(r, r.Client, machineContext.GetObjectMeta())
+	machine, err := clusterutilv1.GetOwnerMachine(ctx, r.Client, machineContext.GetObjectMeta())
 	if err != nil {
 		return reconcile.Result{}, err
 	}
 	if machine == nil {
-		logger.V(2).Info("waiting on Machine controller to set OwnerRef on infra machine")
+		log.Info("Waiting on Machine controller to set OwnerRef on VSphereMachine")
 		return reconcile.Result{}, nil
 	}
 
```

Comment:
Sorry, I might have caused some confusion here :D

Let's add the machine to the logger and context here, so we have it already set for inside `fetchCAPICluster` and don't have to set it in there.


```suggestion

	log := log.WithValues("Machine", klog.KObj(machine))
	ctx = ctrl.LoggerInto(ctx, log)


```